### PR TITLE
Add purge cache button

### PR DESCRIPTION
**Pull request by yuxitang-amzn**
*Issue #, if available:*

*Description of changes:*
Add purge cache button. This is a feature parity to security plugin before 1.10.

image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by bandinib-amzn — APPROVED** [review]

File: public/apps/configuration/panels/get-started.tsx [hunk 1]
```diff
@@ -214,6 +214,29 @@ export function GetStarted(props: AppDependencies) {
...
+
+        <EuiSpacer size="l" />
+
+        <EuiPanel paddingSize="l">
+          <EuiTitle size="s">
+            <h3>Optional: Purge cache</h3>
+          </EuiTitle>
+          <EuiText size="s" color="subdued">
+            <p>
+              By default, the security plugin caches authenticated uses, along with their roles and
```
Comment (tianleh) [inline public/apps/configuration/panels/get-started.tsx:226]: Typo: uses-> users

File: public/apps/configuration/panels/get-started.tsx [hunk 2]
```diff
@@ -214,6 +214,29 @@ export function GetStarted(props: AppDependencies) {
...
+              permissions. This option will purge cached users, roles and permissions.
```
Comment (tianleh) [inline public/apps/configuration/panels/get-started.tsx:227]: Just double check: tenant is not related.

Comment (yuxitang-amzn) [reply public/apps/configuration/panels/get-started.tsx:227]: Nope only information from external IDP are refreshed.

Comment (tianleh) [reply public/apps/configuration/panels/get-started.tsx:227]: In this case, shall we make it clear in the message? I mean the external IDP part.

File: public/apps/configuration/panels/get-started.tsx (hunk 1 above)
Comment (yuxitang-amzn) [reply public/apps/configuration/panels/get-started.tsx]: Fixed.

File: public/apps/configuration/panels/get-started.tsx (hunk 2 above)
Comment (yuxitang-amzn) [reply public/apps/configuration/panels/get-started.tsx:227]: I don't think we need to go into that detail.

**Review by tianleh — APPROVED** [review]

**Pull request merged**